tests/test_browser.py

```diff
@@ -3814,18 +3823,21 @@ def test_pthread_call_sync_on_main_thread(self):
 
   # Test that it is possible to asynchronously call a JavaScript function on the main thread.
   @requires_threads
+  @no_wasm_backend('TODO - fix final pthreads tests')
   def test_pthread_call_async_on_main_thread(self):
     self.btest(path_from_root('tests', 'pthread', 'call_async_on_main_thread.c'), expected='7', args=['-O3', '-s', 'USE_PTHREADS=1', '-s', 'PROXY_TO_PTHREAD=1', '-DPROXY_TO_PTHREAD=1', '--js-library', path_from_root('tests', 'pthread', 'call_async_on_main_thread.js')])
     self.btest(path_from_root('tests', 'pthread', 'call_async_on_main_thread.c'), expected='7', args=['-O3', '-s', 'USE_PTHREADS=1', '-DPROXY_TO_PTHREAD=0', '--js-library', path_from_root('tests', 'pthread', 'call_async_on_main_thread.js')])
     self.btest(path_from_root('tests', 'pthread', 'call_async_on_main_thread.c'), expected='7', args=['-Oz', '-DPROXY_TO_PTHREAD=0', '--js-library', path_from_root('tests', 'pthread', 'call_async_on_main_thread.js')])
 
   # Tests that spawning a new thread does not cause a reinitialization of the global data section of the application memory area.
+  @no_wasm_backend('no memory init options')
```

Comment:
Does the memory init file not still work with the wasm backend?

Reply:
Ah yes, we did add that, thanks. Enabling.